README.md

```diff
@@ -56,6 +57,9 @@ VS Code collects usage data and sends it to Microsoft to help improve our produc
 
 ## Change Log
 
+Version 0.3.9 (02/01/2018)
+- Add JSON outline for ARM depolyment templates
```

Comment:
This is the first version with the JSON outline? That seems like a pretty big feature to me - warranting a minor version bump